MAINTAINERS.yml

```diff
@@ -1841,6 +1838,489 @@ West:
+  labels:
+    - manifest-hal_nordic
+
+"West project: hal_nuvoton":
+  status: maintained
+  maintainers:
+    - MulinChao
+    - ChiHuaL
+    - WealianLiao
+  collaborators:
+    - MulinChao
+    - ChiHuaL
+    - WealianLiao
+    - sjg20
+    - keith-zephyr
+    - jackrosenthal
+    - fabiobaltieri
+    - yperess
+  files:
+    - modules/Kconfig.nuvoton
+  labels:
+    - manifest-hal_nuvoton
+
+"West project: hal_nxp":
+  status: maintained
+  maintainers:
+    - dleach02
+  collaborators:
+    - mmahadevan108
+    - danieldegrasse
+  files:
+    - modules/hal_nxp/
+    - modules/Kconfig.imx
+    - modules/Kconfig.mcux
+  labels:
+    - manifest-hal_nxp
+
+"West project: hal_openisa":
+  status: odd fixes
+  collaborators:
+    - dleach02
+  files:
+    - modules/Kconfig.vega
+  labels:
+    - manifest-hal_openisa
+
+"West project: hal_quicklogic":
+  status: odd fixes
+  collaborators:
+    - kowalewskijan
+  files: []
+  labels:
+    - manifest-hal_quicklogic
+
+"West project: hal_rpi_pico":
+  status: maintained
+  maintainers:
+    - yonsch
+  files:
+    - modules/hal_rpi_pico/
+  labels:
+    - manifest-hal_rpi_pico
+
+"West project: hal_silabs":
+  status: odd fixes
+  collaborators:
+    - yonsch
+    - mnkp
+    - chrta
+  files:
+    - modules/Kconfig.silabs
+  labels:
+    - manifest-hal_silabs
+
+"West project: hal_st":
+  status: maintained
+  maintainers:
+    - avisconti
+  collaborators:
+    - erwango
+  files:
+    - modules/Kconfig.st
+  labels:
+    - manifest-hal_st
+
+"West project: hal_stm32":
+  status: maintained
+  maintainers:
+    - erwango
+  collaborators:
+    - FRASTM
+    - ABOSTM
+  files:
+    - modules/Kconfig.stm32
+  labels:
+    - manifest-hal_stm32
+
+"West project: hal_telink":
+  status: maintained
+  maintainers:
+    - yurvyn
+  files:
+    - modules/Kconfig.telink
+  labels:
+    - manifest-hal_telink
+
+"West project: hal_ti":
+  status: odd fixes
+  collaborators:
+    - cfriedt
+    - gmarull
+  files: []
+  labels:
+    - manifest-hal_ti
+
+"West project: hal_xtensa":
+  status: maintained
+  maintainers:
+    - dcpleung
+  collaborators:
+    - andyross
+    - nashif
+  files:
+    - modules/Kconfig.xtensa
+  labels:
+    - manifest-hal_xtensa
+
+"West project: libmetal":
+  status: odd fixes
+  collaborators:
+    - carlocaione
+    - arnopo
+  files:
+    - modules/Kconfig.libmetal
+  labels:
+    - manifest-libmetal
+
+"West project: liblc3codec":
+  status: maintained
+  maintainers:
+    - Casper-Bonde-Bose
+  collaborators:
+    - thalley
+    - asbjornsabo
+  files:
+    - modules/liblc3codec/
+  labels:
+    - manifest-liblc3codec
+
+"West project: littlefs":
+  status: odd fixes
+  files:
+    - modules/littlefs/
+  labels:
+    - manifest-littlefs
+
+"West project: loramac-node":
+  status: maintained
+  maintainers:
+    - Mani-Sadhasivam
+  files:
+    - modules/loramac-node/
+  labels:
+    - manifest-loramac-node
+
+"West project: lvgl":
+  status: odd fixes
+  files:
```

Comment:
@brgl would you like to maintain or be added as a collaborator?

Reply:
Sure, would be great. I would also love to be able to add labels to PRs and launch CI workflows for first-time contributors in the main zephyr tree if possible.

Reply:
Are display collaborators not asked in this regard?

Reply:
> Are display collaborators not asked in this regard?

@jfischer-no If you are volunteering for the role, please say so clearly and I will add you

Reply:
> Sure, would be great. I would also love to be able to add labels to PRs and launch CI workflows for first-time contributors in the main zephyr tree if possible.

@brgl awesome! Please read https://docs.zephyrproject.org/latest/project/project_roles.html#contributor and nominate yourself for Triage role as Contributor or as Collaborator as you see fit.